### PR TITLE
Feature/funfact card

### LINKED ISSUE #3: Create fun fact card

### PR DESCRIPTION
**Pull request by sigbbe**
Closes #3

### CODE REVIEW

File: src/components/Scoreboard.tsx [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
 import { HighscoreEntry } from '../types/game';
+import FunfactCard from './funfact-card';
+
+// const rank = highscores.indexOf(highscore) + 1;
+//                         return (
+//                             <Tr key={rank}>
+//                                 <Td>{rank + '.'}</Td>
+//                                 <Td>{highscore.name}</Td>
+//                                 <Td>{highscore.score}</Td>
+//                             </Tr>
```
Comment (Jorrre) [inline src/components/Scoreboard.tsx:13]: Remove this

File: src/components/Scoreboard.tsx [hunk 2]
```diff
@@ -21,32 +21,32 @@ function Scoreboard() {
...
         updateHighscores();
     }, []);
+    const listItems = highscores.map((item, i) => (
+        <ListItem
+            w={'65%'}
+            margin={'auto'}
+            fontSize={'1.1rem'}
+            listStylePos={'inside'}
+            textAlign={'left'}
+            key={`item-${item.name}`}>
```
Comment (Jorrre) [inline src/components/Scoreboard.tsx:31]: Could lead to duplicate keys, use i instead

File: src/styles/Theme.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+const theme = extendTheme({
+    colors: {
+        brand: {
+            50: '#44337A',
+            100: '#B794F4',
+            500: '#B794F4', // you need this
+        },
+    },
+});
```
Comment (Jorrre) [inline src/styles/Theme.ts:11]: Same as in main.ts

File: src/main.tsx [hunk 4]
```diff
@@ -1,13 +1,26 @@
...
 
+const theme = extendTheme({
+    colors: {
+        brand: {
+            50: '#44337A',
+            100: '#B794F4',
+            500: '#B794F4', // you need this
+        },
+    },
+});
```
Comment (Jorrre) [inline src/main.tsx:16]: Same as in Theme.ts?

File: src/pages/solarpanel/SolarPanelPage.tsx [hunk 5]
```diff
@@ -0,0 +1,68 @@
+import { Box } from '@chakra-ui/react';
+import { Canvas, useFrame } from '@react-three/fiber';
+import { FC, useEffect, useRef, useState } from 'react';
+import * as THREE from 'three';
+
+function Skrt(props) {
```
Comment (Jorrre) [inline src/pages/solarpanel/SolarPanelPage.tsx:6]: Skrt! Also untyped props

File: src/main.tsx (hunk 4 above)
Comment (sigbbe) [reply src/main.tsx:16]: Yes, I will fix it now :)

**Review by Jorrre — APPROVED** [review]
LGTM

**Pull request merged**